### PR TITLE
adds syndie commander hardsuit modkit

### PR DESCRIPTION
**Pull request by SonixApache**
boss

🆑 
 - rscadd: Nuke op leaders now get a special rigsuit modkit for showing off who's boss.

### CODE REVIEW

File: code/game/objects/items/devices/modkit.dm [hunk 1]
```diff
@@ -111,6 +111,20 @@
...
 
+/obj/item/device/modkit/syndi_commander
+	name = "syndicate commander hardsuit modification kit"
+	desc = "For showing who's the boss. Apply to hardsuit."
+
+/obj/item/device/modkit/syndi_commander/New()
+	..()
+	parts = new/list(1)
+	original = new/list(1)
+	finished = new/list(1)
```
Comment (jknpj) [inline code/game/objects/items/devices/modkit.dm]: you can do all these as new/list(1 = /obj/item/clothing/suit/space/rig/syndi/commander)

Comment (ShiftyRail) [reply code/game/objects/items/devices/modkit.dm]: `new` isn't needed in this context, also I'm not sure if the `1 = typepath` is really necessary

Comment (SonixApache) [reply code/game/objects/items/devices/modkit.dm]: i hate oldcoders so much

Comment (jknpj) [reply code/game/objects/items/devices/modkit.dm]: fix shitcode

Comment (SonixApache) [reply code/game/objects/items/devices/modkit.dm]: why did you guys bitch at me and not at @Kurfursten when he did the exact same thing in #24969 😔

**Comment by SonixApache** [comment]
image
fixed, tested, gib merge

File: code/game/objects/items/devices/modkit.dm (hunk 1 above)
Comment (jknpj) [reply code/game/objects/items/devices/modkit.dm]: This whole file needs a nuclear genocide.

**Pull request merged**